src/app/beta/orgs/[orgId]/canvassassignments/[canvassAssId]/assignees/route.ts

```diff
@@ -0,0 +1,39 @@
+import mongoose from 'mongoose';
+import { NextRequest } from 'next/server';
+
+import asOrgAuthorized from 'utils/api/asOrgAuthorized';
+import { CanvassAssigneeModel } from 'features/areas/models';
+import { ZetkinCanvassAssignee } from 'features/areas/types';
+
+type RouteMeta = {
+  params: {
+    canvassAssId: string;
+    orgId: string;
+  };
+};
+
+export async function GET(request: NextRequest, { params }: RouteMeta) {
+  return asOrgAuthorized(
+    {
+      orgId: params.orgId,
+      request: request,
+      roles: ['admin', 'organizer'],
+    },
+    async () => {
+      await mongoose.connect(process.env.MONGODB_URL || '');
+
+      const canvassAssigneeModels = await CanvassAssigneeModel.find({
+        canvassAssId: params.canvassAssId,
```

Comment:
Maybe `orgId` here as well?

Reply:
I don't think this model is used anymore, right? But even if it was I wouldn't generally put `orgId` on things whose relationship to an organization can still be derived through some secondary object (in this case the assignment).